docs/antrea-ipam.md

```diff
@@ -218,6 +218,34 @@ metadata:
 A StatefulSet Pod's IP will be kept after Pod restarts, when the IP is allocated from the
 annotated IPPool.
 
+### IPAM behaviors
+
+When `AntreaIPAM` is enabled, `antrea-controller` will watch IPPool CRs and StatefulSets
+from K8s apiserver, and `antrea-agent` will register IPAM driver to CNI server.
+
+#### On IPPool CR create/update event
+
+antrea-controller will update IPPool usage.
```

Comment:
Better to keep the format consistent to wrap it as `antrea-controller`
It's vague about IPPool usage, do you mean the IPPool's status about its usage?

Reply:
Thanks. Updated this line to
`antrea-controller` will check annotations and preallocate IPs from the IPPool.